src/commands/createRepo/RepoCreateStep.ts

```diff
@@ -53,10 +54,20 @@ export class RepoCreateStep extends AzureWizardExecuteStep<IStaticWebAppWizardCo
             ext.outputChannel.appendLog(pushedBranch);
             progress.report({ message: pushedBranch });
 
-            wizardContext.branchData = (await client.repos.getBranch({ repo: newRepoName, owner: nonNullProp(wizardContext, 'orgData').login, branch: nonNullProp(branch, 'name') })).data;
+            // getBranch will return undefined sometimes, most likely a timing issue so try to retrieve it for a minute
+            const maxTimeout = Date.now() + 60 * 1000;
```

Comment:
Any time we add "retry" logic I think we should add telemetry to see if the retry is actually helping. Basically, there's three cases to watch out for:
1. It succeeds on the first attempt
2. It succeeds on a retry
3. It times out

If only 1 and 3 are happening in telemetry, we know we can remove the retry logic to save users in case 3 from waiting a minute for no reason.